addons/website/views/snippets/s_numbers_charts.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_numbers_charts" name="Numbers Charts">
+    <section class="s_numbers_charts pt48 pb48">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="12">
+                <div class="o_grid_item g-col-lg-6 g-height-6" style="grid-area: 1 / 1 / 6 / 7; z-index: 1">
+                    <h3>Key Metrics of Company's<br/>Achievements</h3>
+                    <p class="lead">Our key metrics, from revenue growth to customer retention and market expansion, highlight our strategic prowess and commitment to sustainable business success.</p>
+                </div>
+                <div class="o_grid_item g-col-lg-5 g-height-3" style="grid-area: 6 / 1 / 9 / 6; z-index: 3">
+                    <h4 class="display-4">$ 32M</h4>
+                    <p>Our clients have saved $32 million using our services.</p>
+                    <div class="s_progress_bar s_progress_bar_label_hidden" data-display="inline" data-vcss="001">
```

Comment:
I think that this needs `data-snippet="s_progress_bar"` 😉
(check similar occurrences as well)